All project code/index.js

```diff
@@ -371,7 +398,7 @@ async function searchQuery(locationInput) {
             "temperature",
             "precipitation 24 hours"
         ],
-        dataFormat: "json",
+        dataFormat: "html",
```

Comment:
Good change, this will be helpful for presenting data to the user before we have a complete results page done.